<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDivisions</class>
 <widget class="QDialog" name="ViewDivisions">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxA">
     <property name="title">
      <string>Gruppe A</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewA">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxB">
     <property name="title">
      <string>Gruppe B</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBoxC">
     <property name="title">
      <string>Gruppe C</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewC"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QGroupBox" name="groupBoxD">
     <property name="title">
      <string>Gruppe D</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewD"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxE">
     <property name="title">
      <string>Gruppe E</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewE"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxF">
     <property name="title">
      <string>Gruppe F</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewF"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBoxG">
     <property name="title">
      <string>Gruppe G</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewG"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QGroupBox" name="groupBoxH">
     <property name="title">
      <string>Gruppe H</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewH"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxI">
     <property name="title">
      <string>Gruppe I</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewI"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxJ">
     <property name="title">
      <string>Gruppe J</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewJ"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBoxK">
     <property name="title">
      <string>Gruppe K</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewK"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QGroupBox" name="groupBoxL">
     <property name="title">
      <string>Gruppe L</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <widget class="QTableView" name="tableViewL"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
